x/slashrefund/keeper/deposit_test.go

```diff
@@ -3,47 +3,19 @@ package keeper_test
 import (
 	"testing"
 
-	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/made-in-block/slash-refund/x/slashrefund/keeper"
 	"github.com/made-in-block/slash-refund/x/slashrefund/testslashrefund"
 	"github.com/made-in-block/slash-refund/x/slashrefund/types"
 	"github.com/stretchr/testify/require"
 )
 
-func createNDeposit(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.Deposit {
-	items := make([]types.Deposit, n)
-	for i := range items {
-		depPubk := secp256k1.GenPrivKey().PubKey()
-		depAddr := sdk.AccAddress(depPubk.Address())
-		valPubk := secp256k1.GenPrivKey().PubKey()
-		valAddr := sdk.ValAddress(valPubk.Address())
-		items[i].DepositorAddress = depAddr.String()
-		items[i].ValidatorAddress = valAddr.String()
-		items[i].Shares = sdk.ZeroDec()
-		keeper.SetDeposit(ctx, items[i])
-	}
-	return items
-}
-
-func createNDepositForValidator(keeper *keeper.Keeper, ctx sdk.Context, n int) ([]types.Deposit, sdk.ValAddress) {
-	items := make([]types.Deposit, n)
-	valPubk := secp256k1.GenPrivKey().PubKey()
-	valAddr := sdk.ValAddress(valPubk.Address())
-	for i := range items {
-		depPubk := secp256k1.GenPrivKey().PubKey()
-		depAddr := sdk.AccAddress(depPubk.Address())
-		items[i].DepositorAddress = depAddr.String()
-		items[i].ValidatorAddress = valAddr.String()
-		items[i].Shares = sdk.ZeroDec()
-		keeper.SetDeposit(ctx, items[i])
-	}
-	return items, valAddr
-}
+// -------------------------------------------------------------------------------------------------
+// Test deposit
+// -------------------------------------------------------------------------------------------------
 
 func TestDepositGet(t *testing.T) {
```

Comment:
Should be renamed in TestGetDeposit since the tested function is GetDeposit().

Reply:
right